README.md

```diff
@@ -126,12 +126,87 @@ If they are not set explicitely, most of them have a default setting that will b
-| -------- | ---------------------------- | -------- | ------- | ---- |
-| `template_location` | `TEMPLATING_TEMPLATE_LOCATION` | Yes | | The location of the template that should be used during the evaluation. If this location starts with either `http` or `https` it will be considered a URL and will be loaded from the specified endpoint over the network. In all other case the location is being considered a file on the local file system. |
-| `template_auth_username` | `TEMPLATING_AUTH_USERNAME` | No | | The username to be used when accessing the template via network using HTTP basic auth. |
-| `template_auth_password` | `TEMPLATING_AUTH_PASSWORD` | No | | The password to be used when accessing the template via network using HTTP basic auth. |
-| `content` |  | Yes | | The content to be used. |
-| `content_placeholder` | `TEMPLATING_CONTENT_PLACEHOLDER` | No | `INCLUDE_CONTENT_HERE` | The placeholder within the template that should be replaced with the content from the service. |
-| `title` | `TEMPLATING_TITLE` | No | `BetterDoc` | The title to be used. |
-| `title_placeholder` | `TEMPLATING_TITLE_PLACEHOLDER` | No | `INCLUDE_TITLE_HERE` | The placeholder within the template that should be replaced with the title. |
+| Property | Default environment variable | Required | Description |
+| -------- | ---------------------------- | -------- | ----------- |
+| `template_location` | `TEMPLATING_TEMPLATE_LOCATION` | Yes | The location of the template that should be used during the evaluation. If this location starts with either `http` or `https` it will be considered a URL and will be loaded from the specified endpoint over the network. In all other case the location is being considered a file on the local file system. |
+| `template_auth_username` | `TEMPLATING_AUTH_USERNAME` | No | The username to be used when accessing the template via network using HTTP basic auth. |
+| `template_auth_password` | `TEMPLATING_AUTH_PASSWORD` | No | The password to be used when accessing the template via network using HTTP basic auth. |
+
+## Dialects
+
+The different dialects have additional configuration options on their own
+
+### `MustacheDialect`
+
+The `MustacheDialect` can either be set explicitely (see below) and will be implicitely selected if no dialect has been set explicitely **and** the template path ends with `.mustache` or `.mustache.html`.
+
+#### Usage
+
+Template:
+
+```html
+<body>Hello {{name}}</body>
+```
+
+Ruby code:
+
+```ruby
+engine = Betterdoc::Website::Templating::Engine.new
+engine.template_location = "http://www.example.com/template"
+engine.dialect = Betterdoc::Website::Templating::Dialects::Mustache.new
+result = engine.resolve_content(name: "my_name")
+```
+
+Output:
+
+```html
+<body>Hello my_name</body>
+```
+
+### `SimpleDialect`
+
+The `SimpleDialect` will be selected automatically if no dialect has been selected explicitely and the template name doesn't match the pattern for the `MustacheDialect` (see above).
```

Comment:
We should make clear that this dialect does only support exactly two replacement placeholders with predefined names -- one could think that it follows some convention to do the replacement of different placeholders.

Reply:
I've reorganized the documentation for both dialects and clarified the usage.